### PR TITLE
created node permissions

### PR DESCRIPTION
**Pull request by katamaks**

### CODE REVIEW

**Comment by jandevriesmendix** [comment]
Looks like this should be part of the Developer Portal space.

File: howtogeneral/mendixcloud/node-permissions.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+title: "Node Permissions of Your Licensed App"
+space: ""
+category: ""
+description: "Set a description with a maximum of 140 characters; this should describe what the goal of the document is, and it can be different from the document introduction; this is optional, and it can be removed"
+tags: "[Add a maximum of 5-7 tags/keywords; keep them focused on the most important topics of the document;each tag should have quotation marks and be separated by a comma, for example: "Samba", "MxCloud", "cloud", "share"; the tags should be enclosed with brackets and quotation marks]"
+---
+
+# 1 Introduction
+
+Access Management for your Mendix Cloud environments is handled in the **Node Security** tab. Each team member can subscribe or unsubscribe to the alerts, and the **Technical Contact** can manage the permissions of each team member.
```
Comment (jandevriesmendix) [inline howtogeneral/mendixcloud/node-permissions.md]: Where is this Node Security tab you speak of?

File: howtogeneral/mendixcloud/node-permissions.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+title: "Node Permissions of Your Licensed App"
+space: ""
+category: ""
+description: ""
+tags: ""
+---
+
+## 1 Introduction
+
+Access management for your Mendix Cloud environments is handled in the **Node Security** tab. Each team member can subscribe or unsubscribe to the alerts, and the **Technical Contact** can manage the permissions of each team member.
```
Comment (jandevriesmendix) [inline howtogeneral/mendixcloud/node-permissions.md]: Where is this **Node Security** tab?

File: howtogeneral/mendixcloud/node-permissions.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+![](attachments/settings/nodepermission.jpg)
+
+### 5.1 Transport Rights
+
+With transport rights you can deploy new versions of the application to the node. You can create new deployment packages, stop and start the environment, and change configuration settings such as constants and scheduled events.
+
+### 5.2 Access to Backups
+
+This permission grants access to the backups of the environment. You can view, create, download, and restore a backup.
```
Comment (jtwaleson) [inline howtogeneral/mendixcloud/node-permissions.md]: each action should be a link to the documentation of the operation (create backup, download backup, etc).

File: howtogeneral/mendixcloud/node-permissions.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+### 5.3 Receive Alerts
+
+When **Receives Alerts** is checked, this person will receive an email when an alert is triggered. Alerts are triggered when the app went offline unexpectedly, the application logged a message on level **Critical**, the health check failed, or various infrastructure problems occurred.
+
+### 5.4 API Rights
+
+With the API rights you can use the **Deploy API** to get programmatic access to the environment. Naturally, the API does not require two-factor authentication, so API access is disabled for the production environment by default. The Technical Contact can assign API access for each user. Note that the **API Rights** are needed in addition to the other permissions, so in order to access backups via the API you will need both **Access to Backups** as well as **API Rights**.
```
Comment (jtwaleson) [inline howtogeneral/mendixcloud/node-permissions.md]: Deploy API should be a link to the docs of the deploy api.

File: howtogeneral/mendixcloud/node-permissions.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+### 5.5 Access to Monitoring
+
+With this permission you can view the application metrics, logs, and alerts. These allow you to successfully operate your Mendix Cloud environments.
```
Comment (jtwaleson) [inline howtogeneral/mendixcloud/node-permissions.md]: Link to "Monitoring your cloud environment" or a similar document.

File: howtogeneral/mendixcloud/node-permissions.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+The Technical Contact manages the cloud node and can edit the privileges of regular team members with the **View Deploy and Monitor** permissions. Only the Technical Contact can give his user role to another team member by clicking **Change to Technical Contact**. After this, the new user has the Technical Contact role, the old user does not.
+
+The Technical Contact of the app is the first point of contact for [Mendix Support](https://www.support.mendix.com), and will receive alerts regarding the cloud node.
+
+### 4.1 Alerts
+
+The Technical Contact will receive the following alerts from the cloud node:
+
+*   Notifications for maintenance from **Mendix Support**
+*   Alerts from the node when problems arise (CPU load is high, running out of disk space, etc.). The Technical Contact cannot turn these off
```
Comment (jtwaleson) [inline howtogeneral/mendixcloud/node-permissions.md]: Link to alerts documentation

Comment (katamaks) [inline howtogeneral/mendixcloud/node-permissions.md]: Done: To find more information about **Alerts**, read paragraph **2 Alerting Categories and Thresholds**. of the [Monitoring Application Health](/howtogeneral/mendixcloud/monitoring-application-health) documentation.

**Pull request merged**